tutorindigo/plugin.py

```diff
@@ -119,16 +121,17 @@ def _override_openedx_docker_image(
         (
             "mfe-dockerfile-post-npm-install-authn",
             """
-RUN npm install '@edx/brand@npm:@edly-io/indigo-brand-openedx@^2.0.0'{% if INDIGO_ENABLE_DARK_THEME %} --theme=dark{% endif %}
+RUN npm install '@edx/brand@npm:@edly-io/indigo-brand-openedx@^2.1.0'
 """,
         ),
         # Tutor-Indigo v2.1 targets the styling updates in discussions and learner-dashboard MFE
         # brand-openedx is related to styling updates while others are for header and footer updates
         (
             "mfe-dockerfile-post-npm-install-discussions",
             """
-RUN npm install '@edx/brand@npm:@edly-io/indigo-brand-openedx@^2.0.0'{% if INDIGO_ENABLE_DARK_THEME %} --theme=dark{% endif %}
-RUN npm install '@edx/frontend-component-header@npm:@edly-io/indigo-frontend-component-header@~3.0.0'
+RUN npm install universal-cookie
```

Comment:
I'd rather avoid installing new dependencies -- especially if they are not pinned. universal-cookie is already installed via frontend-platform, so it's safe to assume that it's available in all MFEs, right? If yes, can we remove this dependency here? (and below)

Reply:
Yes, tested today that image build is successful if we don't install universal-cookie. I've removed the package from plugin.py.